src/taipy/core/config/data_node_config.py

```diff
@@ -1030,6 +1052,59 @@ def _configure_mongo_collection(
             id, DataNodeConfig._STORAGE_TYPE_VALUE_MONGO_COLLECTION, scope, validity_period, **properties
         )
 
+    @classmethod
+    def _configure_s3_object(
```

Comment:
This method should also be injected to Config. You can add a line similar to other datanode types as:

https://github.com/Avaiga/taipy-core/blob/a01269fb268fe0001ca6ee27263b77271c53eb83/src/taipy/core/config/__init__.py#L39-L58

and


https://github.com/Avaiga/taipy-core/blob/626ef42c6bc584609134109331915e33c2f59f03/tests/conftest.py#L357-L375